extension/jaegerremotesampling/extension_test.go

```diff
@@ -61,24 +66,57 @@ func TestStartAndShutdownRemote(t *testing.T) {
 	cfg := testConfig()
 	cfg.GRPCServerSettings.NetAddr.Endpoint = "127.0.0.1:0"
 	cfg.Source.Remote = &configgrpc.GRPCClientSettings{
-		Endpoint:     fmt.Sprintf("127.0.0.1:%d", lis.Addr().(*net.TCPAddr).Port),
+		Endpoint: fmt.Sprintf("127.0.0.1:%d", lis.Addr().(*net.TCPAddr).Port),
+		TLSSetting: configtls.TLSClientSetting{
+			Insecure: true, // test only
+		},
 		WaitForReady: true,
+		Headers: map[string]configopaque.String{
```

Comment:
New behavior being tested

Reply:
I think we should do a call with and without the test headers. wdyt?